plugins/scanners/scanoss/src/main/kotlin/ScanOssConfig.kt

```diff
@@ -26,7 +26,8 @@ data class ScanOssConfig(
     @OrtPluginOption(defaultValue = "https://api.osskb.org/")
     val apiUrl: String,
 
-    /** The API key required to authenticate with the ScanOSS server. */
+    /** The API key used to authenticate with the ScanOSS server. */
+    @OrtPluginOption(defaultValue = "")
     val apiKey: String,
```

Comment:
Out of scope: Should this actually be a `Secret` ?

Reply:
Indeed, thanks for noticing.